README.md

```diff
@@ -1,3 +1,99 @@
-# Node.js Express Web Service using standard `node` image
+# Hardening Docker Workloads using Slim CLI
 
-TO-DO
+## The Problem and the Magic 🪄
```

Comment:
I personally love it, but not every engineer out there appreciates the "magical" part of our solution. Let's stick with "The Problem and The Solution"

Reply:
Great to have this picture that gives the reader an almost instant understanding of the hardening benefits. However, we shouldn't probably be repeating the info from the picture in the text. Instead, I'd focus on the actual problem and the solution. Something along the lines of:

Creating high-quality container images is hard. How to choose an optimal base image? How to configure a multistage build to make the end slim? How to keep the vulnerability scanners calm? One must be a true container expert to get it right. However, there might be an alternative way.

Automated container image hardening:

- Build an image FROM your favorite base.
- Instrument the image with our sensors.
- Run an instrumented container "probing" its functionality.
- Build a hardened version of the image using collected intelligence.

(and here your screenshot goes)